multiple-domain/MultipleDomain.php

```diff
@@ -254,18 +261,70 @@ public function settingsFieldsForDomains()
             $fields = $this->getDomainFields(0);
         }
         $fieldsToAdd = $this->getDomainFields('COUNT');
-        echo $fields
-            . '<p><button type="button" class="button multiple-domain-add">'
-            . __('Add domain', 'multiple-domain') . '</button></p>'
-            . '<p class="description">'
-            . __('A domain may contain the port number. If a base URL restriction is set for a domain, '
-            . 'all requests that don\'t start with the base URL will be redirected to the base URL. '
-            . '<b>Example</b>: the domain and base URL are <code>example.com</code> and </code>/base/path</code>, '
-            . 'when requesting <code>example.com/other/path</code> it will be redirected to '
-            . '<code>example.com/base/path</code>. Additionaly, it\'s possible to set a language for each domain, '
-            . 'which will be used to add <code>&lt;link&gt;</code> tags with a <code>hreflang</code> '
-            . 'attribute to the document head.', 'multiple-domain') . '</p>'
-            . '<script type="text/javascript">var multipleDomainFields = ' . json_encode($fieldsToAdd) . ';</script>';
+
```

Comment:
Maybe this change makes sense to pass the code style validation, but it definitely doesn't make sense from the translation perspective. I don't think that having the HTML code mixed with literal strings, as I did, is the best approach, but breaking some phrases will turn translation very hard, or even impossible in some cases.

Take these examples:

> tags with a

or

> when requesting

I'm pretty sure we can find a solution between my messy HTML/literal mix and your suggestions here.